public/languages/i18n.js

```diff
@@ -1,117 +1,67 @@
-let translation = He;
+let langs = {};
+let lang;
+let langDirection;
 
-const setLanguage = selectedLanguage => {
-  localStorage.setItem('language', selectedLanguage);
-};
-
-const setTranslation = language => {
-  if (language === 'He') {
-    translation = He;
-  } else if (language === 'Ar') {
-    translation = Ar;
-  } else if (language === 'Ru') {
-    translation = Ru;
-  } else {
-    translation = En;
+const getLanguage = () => {
+  let lang = getQueryParam('lang');
+  if (lang) {
+    return lang.toLowerCase();
   }
-};
-
-const i18n = text => {
-  if (translation.hasOwnProperty(text)) {
-    return translation[text];
-  } else {
-    return text;
+  lang = localStorage.getItem('language');
+  if (lang) {
+    return lang.toLowerCase();
   }
+  return 'he';
 };
 
-const getDirection = () => {
-  const language = localStorage.getItem('language');
-  if (language === 'He' || language === 'Ar') {
-    return 'rtl';
-  } else {
-    return 'ltr';
-  }
+// Save language in local storage if not already there, update HTML
+const setLanguage = (language) => {
+  lang = language;
```

Comment:
this assignment is not clear. please use better naming i.e `currLang`  / `selectedLang`

Reply:
Good idea